src/L8/game/checkers/checkers_constans.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class CheckerColors(Enum):
```

Comment:
In English Checkers, all pieces must only use the Black squares, nobody is allowed to jump horizontally or vertically, only diagonally (even when capturing pieces).
Knowing this, we can validate, not based on the color on the board, but the direction of the movement. So I think we could do without this Enum.